src/main/java/org/aksw/simba/lemming/metrics/single/edgetriangles/EdgeIteratorMetric.java

```diff
@@ -0,0 +1,83 @@
+package org.aksw.simba.lemming.metrics.single.edgetriangles;
+
+import com.carrotsearch.hppc.cursors.IntCursor;
+import grph.Grph;
+import org.aksw.simba.lemming.ColouredGraph;
+import org.aksw.simba.lemming.metrics.AbstractMetric;
+import org.aksw.simba.lemming.metrics.single.SingleValueMetric;
+import toools.set.IntSet;
+import toools.set.IntSets;
+
+import java.util.*;
+
+/**
+ * @author DANISH AHMED on 7/3/2018
+ */
+public class EdgeIteratorMetric extends AbstractMetric implements SingleValueMetric {
+    private ColouredGraph graph;
```

Comment:
Why does this class have a reference to the graph? Please remove it since it breaks the reusability of the class.